rdr_service/offline/genomic_pipeline.py

```diff
@@ -604,3 +604,28 @@ def load_awn_manifest_into_raw_table(
             raw_job.get('dao'),
             cvl_site_id=cvl_site_id
         )
+
+
+def notify_email_group_of_yes_no_yes_w1il_participants(since_datetime):
+    dao = GenomicQueriesDao()
+    participant_list = dao.get_w1il_yes_no_yes_participants(start_datetime=since_datetime)
+
+    logging.warning(
+        f'The following {len(participant_list)} participants '
+        f'switched back to a Yes consent for GROR after being in W1IL: ' +
+        ','.join([f'{participant.participantId}' for participant in participant_list])
+    )
+
```

Comment:
Should we be checking for the `participant_list` list before further executing the email?

Reply:
Good idea, I'll add that.